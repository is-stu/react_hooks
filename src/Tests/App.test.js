import { shallow } from 'enzyme';
import React from 'react';
import App from '../App';

describe('Test for HookApp component', () => {
  test('Should show it correctly', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
