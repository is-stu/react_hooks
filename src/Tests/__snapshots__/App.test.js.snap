// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for HookApp component Should show it correctly 1`] = `
<Fragment>
  <p>
    hi 
  </p>
</Fragment>
`;
